lightning/src/ln/peer_handler.rs

```diff
@@ -441,10 +453,16 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref> PeerManager<D
 			ephemeral_key_midstate,
 			peer_counter_low: AtomicUsize::new(0),
 			peer_counter_high: AtomicUsize::new(0),
+			peer_connectivity_updater: None,
```

Comment:
Then name this perhaps `connectivity_listener` as the actual value is listening for updates not notifying of updates (which is what `PeerManager` is doing).